src/ProjectTemplates/test/template-baselines.json

```diff
@@ -1,1273 +1,101 @@
 {
-  "razor": {
```

Comment:
I'm deleting the non-`blazorwasm` baselines, since the `blazor-wasm` branch is now only meant to deal with blazorwasm things. The other baselines would fail because we're not building their template packages.

This will lead to a nontrivial reintegration with upstream branches later, but I don't think it can be avoided.